app/blueprints/assessments/routes.py

```diff
@@ -267,10 +273,15 @@ def fund_dashboard(fund_short_name: str, round_short_name: str):
     future_tag_types = thread_executor.submit(get_tag_types)
 
     all_applications_metadata = future_all_applications_metadata.result()
+    applications_assigned_to_user = future_applications_assigned_to_user.result()
     application_overviews = future_application_overviews.result()
     active_fund_round_tags = future_active_fund_round_tags.result()
     tag_types = future_tag_types.result()
 
+    app_ids_assigned_to_user = set(
```

Comment:
```suggestion
    app_ids_assigned_to_user = set(
        application["application_id"] for application in applications_assigned_to_user
    ) if applications_assigned_to_user else set()
```

Reply:
Not using this code anywhere, so I'll mark it as resolved.